services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -250,6 +250,21 @@ type ContentWebsiteSchedule {
   endDate(input: FormatDate = {}): String @momentFormat(localField: "end")
 }
 
+type ContentMagazineSchedule {
+  # fields from magazine.model::Schedule
+  id: ObjectID! @projection(localField: "_id") @value(localField: "_id")
+  publication(input: MagazineSchedulePublicationInput = {}): MagazinePublication! @projection(localField: "product") @refOne(
+    loader: "platformProduct",
+    localField: "product"
+    criteria: "magazinePublication",
+  )
+  issue(input: MagazineScheduleIssueInput = {}): MagazineIssue! @projection @refOne(loader: "magazineIssue")
+  section(input: MagazineScheduleSectionInput = {}): MagazineSection! @projection @refOne(loader: "magazineSection")
+
+  # fields from trait.platform::StatusEnabled
+  status: Int @projection
+}
+
```

Comment:
Pretty sure you don't need to define this; `ContentWebsiteSchedule` was used instead of `WebsiteSchedule` because the source data had different local field mappings (coming from `sectionQuery` instead of the website schedule directly).

You should be able to use `MagazineSchedule` and not define this extra type here